Makefile

```diff
@@ -0,0 +1,85 @@
+#
+# Makefile for vrms-rpm
+# Copyright (C) 2017 Marcin "dextero" Radomski
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License, version 3,
+# as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program (LICENCE.txt). If not, see <http://www.gnu.org/licenses/>.
+#
+
+PREFIX ?= /usr/local
+
+CMD := 'for I in lang/*.po; do echo "build/locale/$$(basename $$I .po)/LC_MESSAGES/vrms-rpm.mo"; done'
+LANGS := $(shell bash -c $(CMD))
+
+CMD := 'for I in man/*.man; do [ "$$I" != "man/en.man" ] && echo "$$(basename "$$I" .man)"; done'
+NON_EN_MANS := $(shell bash -c $(CMD))
+
+help:
+	@echo "TARGETS:"
+	@echo "    build - compile project"
+	@echo "    clean - remove compiled artifacts"
+	@echo "    install - compile & install project"
+	@echo "    remove - uninstall project"
+	@echo ""
+	@echo "VARIABLES:"
+	@echo "    PREFIX - installation prefix (default: /usr/local)"
+
+.PHONY: build
```

Comment:
Having each one separate, just above the target it references, makes it easier to keep .PHONY in sync with actual targets.

If you find one .PHONY variant more elegant, I'll change it.